include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -836,6 +836,29 @@ struct OrtApi {
    */
   ORT_API2_STATUS(ReleaseAvailableProviders, _In_ char **ptr,
                   _In_ int providers_length);
+
+  /**
+     * \param value A tensor created from OrtCreateTensor... function.
+     * \param len element length, not including the trailing '\0' chars.
+     * \param index index of string tensor element, length of element at index will be returned.
+     */
+  ORT_API2_STATUS(GetStringTensorElementLength, _In_ const OrtValue* value, _Out_ size_t* out, size_t index);
+
+  /**
+     * \param s string element contents. The string is NOT null-terminated.
+     * \param value A tensor created from OrtCreateTensor... function.
+     * \param s_len element length, get it from OrtGetStringTensorElementLength.
+     * \param index offset of element of tensor to return.
+     */
+  ORT_API2_STATUS(GetStringTensorElement, _In_ const OrtValue* value, _Out_writes_bytes_all_(s_len) void* s,
+                  size_t s_len, size_t index);
```

Comment:
Hmm, maybe index should be after value in the three APIs to keep it consistent and to move the out parameters towards the end?

Reply:
Same comment as above